### DISCUSSION TITLE
faqs

### DISCUSSION THREAD

**Discussion by giscus** [Eeems-Org/remarkable.guide#60] [General]
# faqs

Is it safe to factory reset my device?, How do I remove all third party software from my device?, Can I downgrade to a different OS version?, How get back to stable after installing the beta?, Can ...

https://remarkable.guide/faqs.html

**Comment by PIAServices** [comment]
After deep discharge of the battery my 'reMarkable 2' refused both the update check and the CloudConnection. Even a reset to the factory settings was unsuccessful.

My solution:

1 - Connect to a PC/MAC via USB --- and then (the order is important)
2 - Activate USB web interface in 'Storage settings' 
3 - Look up, if web interface is reachable via http://10.11.99.1/
4 - Start update process and restart the tablet
5 - Look up if cloud sync is functionally
6 - Re-pairing the account, if necessary

I have summarized the solution in a small blog entry (in German). Feel free to stop by:

https://stromzoo.blog/remarkable-2-verweigert-updatecheck-und-cloud-verbindung/

**Reply by PIAServices** [reply]
Update:

reMarkable support got back to me and pointed out that the error probably occurred due to an incorrectly synchronized clock on the tablet.
That makes sense to me and fits my description that the tablet was unused and unloaded for some time.

The support workaround is similar to the one I found myself:

[...]
It is possible that the internal clock on your tablet is out of sync.

Follow these steps to synchronize the clock on your reMarkable paper tablet:

- Restart your reMarkable.
- Disable the ‘Auto sleep’ function in the settings menu, under Settings > Battery.
- Connect your device to your preferred Wi-Fi network.
- Now connect your device to your PC/Mac with the supplied USB cable.
- Wait for at least 30-40 minutes.
- Try to use the one-time code again.
- When performing these steps, it's important that the computer is also connected to the internet.

In addition, make sure that you don't put the computer or the reMarkable to sleep while waiting.
Both devices have to be "awake" for this to work and for the clocks to be reset properly.
[...]

**Reply by Eeems** [reply]
Thank you for this, I've updated the FAQ with a link to the official article ([Unable to update reMarkable Paper Tablet](https://support.remarkable.com/s/article/Unable-to-update-reMarkable-Paper-Tablet)) that contains these steps.
